tests/include/igl/bbw.cpp

```diff
@@ -8,6 +8,9 @@
 
 TEST(bbw, decimated_knight)
 {
+  #ifdef WIN32
+    return;
```

Comment:
I'm pretty sure this is not the right fix :p

Reply:
this thing was never finishing.
The tests where never run on windows.
My guess it that it comes from the parallel loop

Reply:
this is where it gets stuck:
`parallel_for(m,optimize_weight,2);`

Reply:
Can you try to disable (with an assert) the parallelization inside parallel_for on windows? I had troubles with C++11 threads on VS before.

Reply:
you mean an ifdef and skip the parallel for?